contracts/src/L1/L1ScrollMessenger.sol

```diff
@@ -27,6 +27,11 @@ import {IMessageDropCallback} from "../libraries/callbacks/IMessageDropCallback.
 /// @dev All deposited Ether (including `WETH` deposited throng `L1WETHGateway`) will locked in
 /// this contract.
 contract L1ScrollMessenger is ScrollMessengerBase, IL1ScrollMessenger {
+    /***********
+     * Errors *
+     ***********/
+    error ErrZeroAddress();
```

Comment:
can you move this error to `IScrollMessenger`?

Reply:
done